lib/rubyhub/operations/pull_request/create.rb

```diff
@@ -17,6 +18,15 @@ def call(options)
 
           def data
             @data ||= Rubyhub::Configuration.instance.options.dig(:template, template.to_sym)
+            return unless @data.respond_to?(:to_hash)
+
+            description = @data[:description_main_body]
+            add_description_to_data if description.nil? || description.empty?
```

Comment:
If we read this line, it sounds like `add description to data if this description or description is empty`. It is too strange. I know that you mean add from another place, but naming makes this whole line confusing.